src/models/order.js

```diff
@@ -0,0 +1,33 @@
+import Backbone from 'backbone';
+
+const Order = Backbone.Model.extend({
```

Comment:
Since you're passing a `Quote` instance to the `Order`, you could listen for price changes,

Something like

```
initialize(params) {
  this.listenTo(this.get('quote'), 'change', (quote) => {
    // code to buy a quote, maybe a trigger to alert the view or quote.
  }
},
```